### PR TITLE
[WIP] Implementation of Python workshop using AWS CDK (TypeScript)

### PR DESCRIPTION
**Pull request by dstroppa**
*Issue #, if available:* N/A

*Description of changes:* Implementation of the Python branch using AWS CDK instead of direct use of CloudFormation.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: module-1/README.md [hunk 1]
```diff
@@ -69,81 +80,245 @@ In the terminal, change directory to the newly cloned repository directory:
...
+Now, let's define the infrastructure needed to host our static website.  
 
-This will open `bucket-policy.json` in the File Editor panel.  Replace the string shown with your chosen bucket name used in the previous commands:
+Create a new file called `static-website-stack.ts` in the `lib` folder, and define the skeleton class structure by writing/copying the following code:
 
-![replace-bucket-name.png](/images/module-1/replace-bucket-name.png)
+```typescript
+import cdk = require('@aws-cdk/core');
+
+export class StaticWebsiteStack extends cdk.Stack {
```
Comment (DavidChristiansen) [inline module-1/README.md]: The web app becomes an Angular based SPA and whilst its a static HTML page in module 1, the stack can be reused across all remaining modules.  Consider renaming to web-application-stack.ts as per below?

https://github.com/aws-samples/aws-modern-application-workshop/blob/dotnet-cdk/module-1/cdk/lib/web-application-stack.ts


Should we consider standardising on the naming of the stacks across the branches.

Comment (dstroppa) [reply module-1/README.md]: Got it, will change this.

**Comment by dstroppa** [comment]
> Do we standardise on ‘cdk’ or ‘cdk-complete’ as the name for our code folder?

I've used 'cdk-complete' to differentiate from the 'cdk' folder that will be created as part of the instructions. It also makes it clear that the code in that folder is the completed one.

**Comment by dstroppa** [comment]
> Do we standardise on ‘cdk’ or ‘cdk-complete’ as the name for our code folder?

Also renamed cdk-complete to cdk

**Pull request merged**